lib/resty/cookie.lua

```diff
@@ -117,16 +123,39 @@ function _M.get(self, key)
     return self.cookie_table[key]
 end
 
-function _M.get_all(self)
```

Comment:
let's move the new `get_all` function here. like this, it isn't clear how each function has changed

Reply:
makes sense, moved